processing/update_base_badm.py

```diff
@@ -237,7 +231,9 @@ def driver(self, base_attrs, post_base_only=None):
         badm_map = self.new_db_handler.get_badm_map(psql_conn)
         sites_needing_updates = self.new_db_handler.get_sites_with_updates(
             psql_conn)
-        base_candidate_map = self.flux_db_handler.get_BASE_candidates(
+
+        # Need to revamp
```

Comment:
Consider removing comment unless there is still revamping to be completed?

Reply:
Will do